### ISSUE TITLE
Mouse events interference

### ISSUE THREAD

**Issue by tverilytt** [jadjoubran/Android-Toast#8]
Hi!
I found your Android toast very nice and started to use it. I experienced that it seemed to interfere with mouse event listeners I am using. After some search and trial, it seems like adding this line to your CSS resolves the probelm:

  pointer-events:none;

E.g.

```
#android_toast_container{
  width:100%;
  position:fixed;
  bottom:12%;
  left:0;
  pointer-events:none;
}
```

**Comment by jadjoubran** [comment]
Hi @tverilytt 
Thanks!

Would you mind sending a pull request with this fix?

**Comment by tverilytt** [comment]
Hi! I am somewhat unsure on this. The change will disable mouse events on the toast content. Which might be desireable in some context, and not in others. E.g. if one shows a link in the toast, it should be clickable...

Maybe an option for enabling / disabling pointer events could address this?

What do you think?
-jo

**Comment by jadjoubran** [comment]
hmm makes sense.. we can add to the docs (readme) that if you have an issue similar to yours, please check issue #8
Feel free to submit a PR (you can use github's readme editor)

**Comment by jadjoubran** [comment]
Fixed in #9
